src/Modules/Communications/API/API.Client/README.md

```
@@ -9,9 +9,128 @@ Esta coleção do Bruno contém os endpoints para o módulo de comunicações.
+  "grant_type": "password",
+  "client_id": "meajudaai-api",
+  "username": "admin@meajudaai.com",
+  "password": "admin123"
+}
+```
+
+Copie o `access_token` e configure na variável `{{accessToken}}`.
+
+## 🔧 Variáveis da Collection
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)